fiftyone/utils/paddleocr.py

```
@@ -0,0 +1,418 @@
+    fou.ensure_package(_MIN_TRANSFORMERS)
+
+
+paddleocr = fou.lazy_import("paddleocr", callback=_ensure_paddleocr)
+
+from PIL import Image as PILImage
+
+DEFAULT_DET_MODEL = "PP-OCRv6_medium_det"
+DEFAULT_REC_MODEL = "PP-OCRv6_medium_rec"
+
+
+def _to_numpy_bgr(img):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_to_numpy_bgr`

(ANN202)